src/main/resources/db/migration/V1_5__adding_on_delete_cascade_tables.sql

```diff
@@ -0,0 +1,5 @@
+
+ALTER TABLE response DROP CONSTRAINT user_profile_id_fk1;
+alter table response add constraint user_profile_id_fk1 foreign key (user_profile_id)references user_profile (id) on delete cascade;
+
```

Comment:
why on delete cascade is required on response table?

Reply:
I have added on delete cascade user_profile_id_fk1 constraint of response table which is refering from user_profile table column id.